diff --git a/selfdrive/common/spinner.c b/selfdrive/common/spinner.c
index c571b510..2f22619d 100644
--- a/selfdrive/common/spinner.c
+++ b/selfdrive/common/spinner.c
@@ -21,8 +21,8 @@
 #define SPINTEXT_LENGTH 128
 
 // external resources linked in
-extern const unsigned char _binary_opensans_semibold_ttf_start[];
-extern const unsigned char _binary_opensans_semibold_ttf_end[];
+extern const unsigned char _binary_cheeky_rabbit_ttf_start[];
+extern const unsigned char _binary_cheeky_rabbit_ttf_end[];
 
 extern const unsigned char _binary_spinner_track_png_start[];
 extern const unsigned char _binary_spinner_track_png_end[];
@@ -65,7 +65,7 @@ int spin(int argc, char** argv) {
   NVGcontext *vg = nvgCreateGLES3(NVG_ANTIALIAS | NVG_STENCIL_STROKES);
   assert(vg);
 
-  int font = nvgCreateFontMem(vg, "Bold", (unsigned char*)_binary_opensans_semibold_ttf_start, _binary_opensans_semibold_ttf_end-_binary_opensans_semibold_ttf_start, 0);
+  int font = nvgCreateFontMem(vg, "Bold", (unsigned char*)_binary_cheeky_rabbit_ttf_start, _binary_cheeky_rabbit_ttf_end-_binary_cheeky_rabbit_ttf_start, 0);
   assert(font >= 0);
 
   int spinner_img = nvgCreateImageMem(vg, 0, (unsigned char*)_binary_spinner_track_png_start, _binary_spinner_track_png_end - _binary_spinner_track_png_start);
diff --git a/selfdrive/ui/spinner/Makefile b/selfdrive/ui/spinner/Makefile
index 5e0b3437..b5ddb16e 100644
--- a/selfdrive/ui/spinner/Makefile
+++ b/selfdrive/ui/spinner/Makefile
@@ -48,15 +48,15 @@ spinner: $(OBJS)
            -I$(PHONELIBS)/android_hardware_libhardware/include \
            -c -o '$@' '$<'
 
-opensans_semibold.o: ../themes/base/fonts/opensans_semibold.ttf
+opensans_semibold.o: ../themes/waifu/fonts/cheeky-rabbit.ttf
 	@echo "[ bin2o ] $@"
 	cd '$(dir $<)' && ld -r -b binary '$(notdir $<)' -o '$(abspath $@)'
 
-img_spinner_track.o: ../themes/base/images/spinner_track.png
+img_spinner_track.o: ../themes/waifu/images/spinner_track.png
 	@echo "[ bin2o ] $@"
 	cd '$(dir $<)' && ld -r -b binary '$(notdir $<)' -o '$(abspath $@)'
 
-img_spinner_comma.o: ../themes/base/images/spinner_comma.png
+img_spinner_comma.o: ../themes/waifu/images/spinner_comma.png
 	@echo "[ bin2o ] $@"
 	cd '$(dir $<)' && ld -r -b binary '$(notdir $<)' -o '$(abspath $@)'
 
diff --git a/selfdrive/ui/spinner/spinner b/selfdrive/ui/spinner/spinner
old mode 100644
new mode 100755
index 74f4a394..d171b537
Binary files a/selfdrive/ui/spinner/spinner and b/selfdrive/ui/spinner/spinner differ
